foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,80 @@
+1. Test that your function returns what you expect using `console.log` or [the browser developer tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) before advancing to the next step.
+
+#### Step 3: Play round function
+
+Your game will be played round by round. You will make a function that takes the human and computer player choices as arguments, plays a single round and returns a winner announcement:
+
+1. Create a new function named `playRound`.
+1. Define two parameters for `playRound`: `playerChoice` and `computerChoice`. Use these two parameters to take the human and computer choices as arguments.
+1. Make your function's `playerSelection` parameter case-insensitive so that players can input "rock", "ROCK", "RocK", or other variations.
+1. Write the code for your `playRound` function to `return` a string value representing the round winner, such as: "You lose! Paper beats Rock".
+
+Example code:
+
+```javascript
+function playRound(playerChoice, computerChoice) {
+  // your code here!
+}
+
+const playerSelection = "rock";
+const computerSelection = getComputerChoice();
 
-1. Start a new Git repo for your project.
-1. Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file). This game is going to be played completely from the console, so don't worry about putting anything else in there.
-1. Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'. We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-1. Write a function that plays a single round of Rock Paper Scissors. The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner or tie of the round like so: `"You Lose! Paper beats Rock"`
+console.log(playRound(playerSelection, computerSelection));
+```
 
-   - Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
+#### Step 4: Play game function
 
-1. **Important note:** you want to `return` the results of this function call, *not* `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
+Your game will play 5 rounds. Write a function named `playGame` that uses `playRound` to play 5 rounds, keeps score, and declares a winner at the end:
 
-   ```javascript
-   function playRound(playerSelection, computerSelection) {
-     // your code here!
-   }
+1. Create a new function named `playGame`.
+1. Get the human player choice using the `prompt` method. Read this [MDN article about the prompt method](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) learn more about it.
```

Comment:
Could also combine the link with the prompt method text:

```suggestion
1. Get the human player choice using the [`prompt` method](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt).
```

Reply:
I'm not sure about combining both as it looks are odd:
![image](https://github.com/TheOdinProject/curriculum/assets/36206278/bdf755c4-438a-4afd-8c04-c28a830a52b5)

Reply:
Agreed that visually, at least for 2 word links (1 in backticks, 1 not), it really is odd to look at, and can easily be misinterpreted at 2 separate links at a glance.
At least with more words, the ambiguity is less.

Reply:
Good call, I'm fine with it as-is then